VerbalExpressions/VerbalExpressions.swift

```diff
@@ -6,13 +6,15 @@
 //  Copyright (c) 2014 Dominique d'Argent. All rights reserved.
 //
 
+//: Playground - noun: a place where people can play
```

Comment:
Sorry, was using a playground to type and test it out. Will get rid of it in a new commit.